Dockerfile.ubuntu

```diff
@@ -60,6 +49,3 @@ RUN pip install -r /opt/file-metadata/test-requirements.txt
 
 # Install package
 RUN pip install /opt/file-metadata --upgrade
-
-# Test file-metadata import
-RUN python -c'import file_metadata; print(file_metadata.__version__)'
```

Comment:
The question is not whether file-metadata install works (that does travis - agreed), the question is whether the Dockerfile is valid! Imagine you forgot to add a package in Dockerfile (e.g. libmagickwand-dev).